### DISCUSSION TITLE
Wifi Direct

### DISCUSSION THREAD

**Discussion by chris1111** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#133] [General]
https://blog.immersed.team/wi-fi-direct-8ec23c74fdab

![스크린샷 2021-10-03 오후 10 05 41](https://user-images.githubusercontent.com/38493722/135754911-627dc7e7-2347-4a6d-a22b-a57428095e2e.png)
![스크린샷 2021-10-03 오후 10 05 33](https://user-images.githubusercontent.com/38493722/135754923-ac03a0ae-f63c-497c-9f0e-47ae81d20ca0.png)
![스크린샷 2021-10-03 오후 10 06 40](https://user-images.githubusercontent.com/38493722/135754941-879934b9-6569-40f6-999d-f053c2c3a9b7.png)

I actually tested it, but i don't think mac internet sharing options doesn't works.
I don't know if I did it right. I turned off the SIP,
entered the command well, and the simple Wi-Fi connection works well.
Only internet sharing doesn't work...

It works well with connect Wi-Fi through a program other than the existing Mac Wi-Fi function.

![스크린샷 2021-10-03 오후 10 16 07](https://user-images.githubusercontent.com/38493722/135755348-059cea5a-821c-418e-beb2-c072dac011ba.png)
![스크린샷 2021-10-03 오후 10 16 11](https://user-images.githubusercontent.com/38493722/135755353-646848b2-6f2e-41bf-928a-0ea42c103a56.png)

https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/discussions/146

**Comment by jayala99** [comment]
WL-WN692A3 en Big Sur 11.6.6 iMac 2012 Funcionando correctamente

**Comment by wjimenez27** [comment]
> https://blog.immersed.team/wi-fi-direct-8ec23c74fdab
> 
> ![스크린샷 2021-10-03 오후 10 05 41](https://user-images.githubusercontent.com/38493722/135754911-627dc7e7-2347-4a6d-a22b-a57428095e2e.png) ![스크린샷 2021-10-03 오후 10 05 33](https://user-images.githubusercontent.com/38493722/135754923-ac03a0ae-f63c-497c-9f0e-47ae81d20ca0.png) ![스크린샷 2021-10-03 오후 10 06 40](https://user-images.githubusercontent.com/38493722/135754941-879934b9-6569-40f6-999d-f053c2c3a9b7.png)
> 
> I actually tested it, but i don't think mac internet sharing options doesn't works. I don't know if I did it right. I turned off the SIP, entered the command well, and the simple Wi-Fi connection works well. Only internet sharing doesn't work...
> 
> It works well with connect Wi-Fi through a program other than the existing Mac Wi-Fi function.
> 
> ![스크린샷 2021-10-03 오후 10 16 07](https://user-images.githubusercontent.com/38493722/135755348-059cea5a-821c-418e-beb2-c072dac011ba.png) ![스크린샷 2021-10-03 오후 10 16 11](https://user-images.githubusercontent.com/38493722/135755353-646848b2-6f2e-41bf-928a-0ea42c103a56.png)
> 
> [chris1111/Wireless-USB-Big-Sur-Adapter#146](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/discussions/146)
